README.md

```diff
@@ -23,38 +23,41 @@ Optional prerequisites can be skipped for reduced functionality.
-|                        | `EMAIL_HOST_PASSWORD`   | The SMTP password                                                   | _`Empty string`_                                        |
-|                        | `EMAIL_PORT`            | The SMTP port                                                       | `587`                                                   |
-| External Services      | `CONFIGDB_URL`          | The url to the configuration database                               | `http://localhost`                                      |
-|                        | `DOWNTIMEDB_URL`        | The url to the downtime database                                    | `http://localhost`                                      |
-|                        | `ELASTICSEARCH_URL`     | The url to the elasticsearch cluster                                | `http://localhost`                                      |
-| Task Scheduling        | `DRAMATIQ_BROKER_HOST`  | The broker host for dramatiq                                        | `redis`                                                 |
-|                        | `DRAMATIQ_BROKER_PORT`  | The broker port for dramatiq                                        | `6379`                                                  |
+|                        | Variable                         | Description                                                                                                                                                                 | Default                                                 |
+| ---------------------- | -------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------- |
+| General                | `DEBUG`                          | Whether the application should run using Django's debug mode                                                                                                                | `False`                                                 |
+|                        | `SECRET_KEY`                     | The secret key used for sessions                                                                                                                                            | _`random characters`_                                   |
+|                        | `CSRF_TRUSTED_ORIGINS`           | Comma separated list of trusted origins allowed for CSRF                                                                                                                    | `None`
+|                        | `REQUESTGROUP_DATA_DOWNLOAD_URL` | The url where a user can download requestgroup data. Optionally include `{requestgroup_id}` in the string which will be filled in with the ID of the specific requestgroup. | _`Empty string`_ |
```

Comment:
This is the new setting. All the other changes in the README are changes to the spacing of the table.